README.md

```diff
@@ -0,0 +1,63 @@
+## About ##
+
+Privly is a developing set of browser extensions for protecting content wherever it is posted on the internet. This extension allows users to view content on any website, without the host site being able to read the content. For more information on what Privly is, [read about us](https://priv.ly/pages/about).
+
+## Development Status ##
+
+**Alpha Version**
+
+[![Build Status](https://travis-ci.org/privly/privly-safari.svg)](https://travis-ci.org/privly/privly-safari)
+
+The Privly Safari extension is currently maintained by [Sean McGregor](https://github.com/smcgregor) and [Sambuddha Basu](https://github.com/sammyshj), on behalf of the [Privly Foundation](http://www.privly.org).
+
+## About this Extension ##
+
+This extension currently supports:
+
+* Locally served posting and injectable applications.
+* All the implemented [Injectable Applications](https://github.com/privly/privly-organization/wiki/Injectable-Applications) of the content server.
+
+## Testing/Submitting Bugs ##
+
+Extension integration test cases are found at [test.privly.org](http://test.privly.org). If you have discovered a bug, only [open a public issue](https://github.com/privly/privly-safari/issues/new) on GitHub if it could not possibly be a security related bug. If the bug affects the security of the system, please report it privately at [privly.org](http://www.privly.org/content/bug-report). We will then fix the bug and follow a process of responsible disclosure.
```

Comment:
The bug reporting practices changed for security bugs. Follow the link for the current details.